lib/engine/game/g_1856.rb

```diff
@@ -294,6 +295,7 @@ def create_destinations(destinations)
           dest_arr.each do |d|
             hex_by_id(d).original_tile.icons << Part::Icon.new("../logos/1856/#{corp}")
           end
+          @destinations[corp] = [end_b, end_a].freeze
```

Comment:
this seems to indicate that end_a and end_b are just single elements

Reply:
The cleanup of this method should make it more clear the second side could be an array